News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/async_tasks/RssItemToHtmlTask.java

```diff
@@ -134,7 +136,32 @@ public static String getHtmlPage(RequestManager glide, RssItem rssItem, boolean
         }
 
         String description = rssItem.getBody();
-        if(description.isEmpty() && rssItem.getMediaDescription() != null) {
+
+        if (!description.isEmpty()) {
+            // UUID is used so there is only a very small chance that the placeholder text actually exists in the article
+            var uuid = UUID.randomUUID().toString();
+
+            // pre-blocks shouldn't have their newlines replaced
+            var matcher = Pattern.compile("<pre>(.*?)</pre>", Pattern.MULTILINE | Pattern.DOTALL).matcher(description);
```

Comment:
I believe this can be a static variable, no? This way the pattern doesn't need to get compiled on each call?

Reply:
You're correct! This has been fixed.